database/dataGeneration.js

```diff
@@ -0,0 +1,51 @@
+const SidebarInfo = require('./SidebarInfo');
+const faker = require('faker');
+const db = require('./index.js');
+const fs = require('fs');
+const file = fs.createWriteStream('./database/generatedData.csv')
+
+const numberOfEntries = 10000000;
+file.write('restaurantID,address,neighborhood,crossStreet,parking,dining,cuisines,hours,phone,website,payment,dress,chef\n');
+
+const populateItems = async(callback) => {
+  const randRange = (min, max) => (Math.floor(Math.random() * (max + 1 - min)) + min);

```

Comment:
I like the idea of wrapping this as a function and just reuse it every time you need it. Probably gonna do that in the future